spec/cassettes/OpenStax_Aws_Stack/return_list_of_stacks.yml

```diff
@@ -0,0 +1,994 @@
+                <StackName>main-qa-distribution</StackName>
+                <DriftInformation>
+                  <StackDriftStatus>NOT_CHECKED</StackDriftStatus>
+                </DriftInformation>
+                <StackStatus>CREATE_COMPLETE</StackStatus>
+              </member>
+              <member>
+                <CreationTime>2018-12-12T20:12:15.208Z</CreationTime>
+                <StackId>arn:aws:cloudformation:us-east-1:373045849756:stack/main-staging-distribution/37559230-fe4a-11e8-b7b6-500c3d441629</StackId>
+                <TemplateDescription>The main &quot;front door&quot; CloudFront distribution</TemplateDescription>
+                <StackName>main-staging-distribution</StackName>
+                <DriftInformation>
+                  <StackDriftStatus>NOT_CHECKED</StackDriftStatus>
+                </DriftInformation>
+                <StackStatus>CREATE_COMPLETE</StackStatus>
+              </member>
+              <member>
+                <CreationTime>2018-11-29T13:51:40.243Z</CreationTime>
+                <StackId>arn:aws:cloudformation:us-east-1:373045849756:stack/rex-s3-primary-bucket/e5519090-f3dd-11e8-b7b0-124d1eab42ba</StackId>
+                <TemplateDescription>Primary Rex bucket
+        </TemplateDescription>
+                <StackName>rex-s3-primary-bucket</StackName>
+                <DriftInformation>
+                  <StackDriftStatus>NOT_CHECKED</StackDriftStatus>
+                </DriftInformation>
+                <StackStatus>UPDATE_COMPLETE</StackStatus>
+                <LastUpdatedTime>2018-12-02T21:18:44.728Z</LastUpdatedTime>
+                <StackStatusReason>Export RexPrimaryBucketName cannot be deleted as it is in use by main-qa-distribution and main-staging-distribution</StackStatusReason>
+              </member>
+              <member>
+                <CreationTime>2018-11-29T13:50:35.472Z</CreationTime>
+                <StackId>arn:aws:cloudformation:us-east-1:373045849756:stack/rex-s3-identities/beb7d200-f3dd-11e8-a46c-500c28b4e4fd</StackId>
+                <TemplateDescription>Identities needed by both primary and replica rex buckets
+        </TemplateDescription>
+                <StackName>rex-s3-identities</StackName>
+                <DriftInformation>
+                  <StackDriftStatus>NOT_CHECKED</StackDriftStatus>
+                </DriftInformation>
+                <StackStatus>CREATE_COMPLETE</StackStatus>
+                <StackStatusReason>Export MainDistributionOai cannot be deleted as it is in use by main-qa-distribution and main-staging-distribution</StackStatusReason>
+              </member>
+            </StackSummaries>
+          </ListStacksResult>
+          <ResponseMetadata>
+            <RequestId>69be84d1-ecfd-477d-b217-15532dd46c72</RequestId>
+          </ResponseMetadata>
+        </ListStacksResponse>
+    http_version: 
+  recorded_at: Tue, 07 Apr 2020 13:31:51 GMT
+recorded_with: VCR 4.0.0
```

Comment:
I appreciate you going for the VCR spec; generally speaking I don't like to leak real stack info into the cassettes.  If #56 is what we move to, maybe we could make a couple of fake stacks in the spec and then test that we can query those?